src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseFieldDefinition.java

```diff
@@ -148,14 +146,6 @@ public void setShowConditon(String showConditon) {
         this.showConditon = showConditon;
     }
 
-    public String getDefaultValue() {
-        return defaultValue;
```

Comment:
CaseFieldDefiniton is going to created from Definition store, may be we need this defaultValue method?

Reply:
As per conversation this is not coming from the definition store and it is not used.